src/utils/index.ts

```diff
@@ -49,7 +49,7 @@ export const wrapErrorHandler = (fn: (rq: Request, rs: Response) => Promise<void
  */
 export const extractAssetMapKey = (relatedUrl: string) => {
   const urlArray = relatedUrl.split(".");
-  return urlArray[urlArray.length - 2] ? urlArray[urlArray.length - 2] : "key";
+  return urlArray[urlArray.length - 2] ? urlArray[urlArray.length - 2] : "asset_key";
```

Comment:
nitpick but, if we are not initializing values for these sort of ternaries in the code I'm fine
```suggestion
  const assetKey = urlArray[urlArray.length - 2];
  return assetKey ? assetKey : "asset_key";
```